lib/trino-metastore/src/main/java/io/trino/metastore/tracing/MetastoreAttributes.java

```diff
@@ -23,13 +23,13 @@ final class MetastoreAttributes
     private MetastoreAttributes() {}
 
     public static final AttributeKey<String> SCHEMA = stringKey("trino.schema");
-    public static final AttributeKey<Long> SCHEMA_RESPONSE_COUNT = longKey("trino.hive.response.schema_count");
+    public static final AttributeKey<Long> SCHEMA_RESPONSE_COUNT = longKey("trino.metastore.response.schema_count");
     public static final AttributeKey<String> TABLE = stringKey("trino.table");
-    public static final AttributeKey<Long> TABLE_RESPONSE_COUNT = longKey("trino.hive.response.table_count");
+    public static final AttributeKey<Long> TABLE_RESPONSE_COUNT = longKey("trino.metastore.response.table_count");
```

Comment:
Not backward compatible

Reply:
Good point, I'll update the release notes